lib/fs/modules/LinuxMount.rb

```diff
@@ -315,19 +315,26 @@ def expandLinks(p)
       #
       fs, lp = getFsPathBase(ncp)
       if fs.fileSymLink?(lp)
-        sl = getSymLink(fs, lp)
-        if sl[0, 1] == '/'
-          cp = sl
-        else
-          cp = File.join(cp, sl)
-        end
+        cp = follow_all_symlinks(fs, lp, cp)
```

Comment:
Every time we traverse to a new symlink, the filesystem, `fs`, can change.
I think `follow_all_symlinks` should be implemented at a higher level, so it can call `getFsPathBase` as needed.

Reply:
Ah. Good catch @roliveri.  I'll look at that.